tests/kustomize_build.sh

```diff
@@ -79,3 +79,61 @@ kpt fn source examples |
   kpt fn sink .
 assert_contains_string configmap_the-map.yaml "app: hello"
 assert_dir_exists knative-serving
+
+testcase "kpt_kustomize_build_fn_plugins"
+mkdir fn
+mkdir -p source/base
+mkdir -p source/kustomize/plugin/
+curl -fsSL -o key.asc https://raw.githubusercontent.com/mozilla/sops/master/pgp/sops_functional_tests_key.asc || echo "couldn't create key.asc"
```

Comment:
what does 'kustomize build' have to do with sops? There's a sops function with its own e2e tests.

Reply:
'kustomize build' may use kpt-functions as plugins.
Before adding `docker-cli` to the dockerfile it wasn't possible.

The intent of the test was just to make sure that this worked now.
I could use any of existing container-based functions (ts or go) to write a case that covers running kpt-fn-based plugins inside kustomize_build function. Since I anyway needed to select one function - it was easier for me to make this test with sops, because I knew it would work.

I had another option: to use kustomize_build function as kustomize plugin inside another kustomize_build function that is ran by kpt was an another example, but I just though it was too artificial. 

I can try to rewrite the test with another function. Or may completely remove the test, since the change is trivial.
Or I can move this test to sops e2e test file.

What option seems reasonable to you?

Reply:
My vote is to completely remove the test.

Larger context: We are in the process of conceptually aligning kpt and kustomize projects. Having 'kustomize build' as a KRM function doesn't really make sense in an e2e flow and will be removed in the near future. 

Filed https://github.com/GoogleContainerTools/kpt/issues/1078

Sorry for the confusion caused here.

Reply:
Thank you for the larger context! That sounds really interesting - looking forward to see what it will be.
Meanwhile - removing the test 👍